tensorflow/core/kernels/mlir_generated/BUILD

```diff
@@ -469,6 +507,28 @@ tf_cuda_cc_test(
     ],
 )
 
+tf_cuda_cc_test(
```

Comment:
Does this belong here? Is it not already part of https://github.com/tensorflow/tensorflow/pull/58045

Reply:
Addressed.